components/Trade/Advanced/TradingPanel/TradingInput.tsx

```diff
@@ -133,14 +132,14 @@ export default styled(({ selectedTracer, className, account }: TIProps) => {
                 <InputSelects amount={order?.amountToPay} price={order?.price} selectedTracer={selectedTracer} />
 
                 {/* Dont display these if it is a limit order*/}
-                {order?.orderType !== 1 ? (
-                    <>
-                        {/* Leverage select */}
-                        <Leverage leverage={order?.leverage ?? 1} />
-                    </>
-                ) : (
-                    <></>
-                )}
+                {/*{order?.orderType !== 1 ? (*/}
```

Comment:
Was this intentionally removed??

Reply:
The new leverage slider will take its place here eventually